frontend/console/e2e/module.spec.ts

```diff
@@ -10,6 +10,5 @@ ftlTest('shows verbs for deployment', async ({ page }) => {
 
   await expect(page).toHaveURL(/\/modules\/echo/)
 
-  await expect(page.getByText('Deployment', { exact: true })).toBeVisible()
-  await expect(page.getByText('Created Deployment dpl-echo')).toBeVisible()
+  await expect(page.getByText('module', { exact: true })).toBeVisible()
```

Comment:
Should we look for like `echo` or something since this should be the echo module? :)

Reply:
thanks for catching that! fixed